### DISCUSSION TITLE
Monthly view in Meal Planner

### DISCUSSION THREAD

**Discussion by ditmarvisser** [mealie-recipes/mealie#1827] [Feature Request]
The current meal planner (if im not mistaken) shows 8 days at a time starting yesterday. Is it possible to have a monthly view, or 4 weeks at a time? That would make meal planning a lot more structured.

Related to #1780

**Comment by grasdk** [comment]
I believe this is solved in the current version as is.

**Comment by boc-the-git** [comment]
I agree with @grasdk's assessment - this is in the product now.
